llama_cpp/server/app.py

```diff
@@ -882,19 +653,16 @@ class ModelList(TypedDict):
 
 @router.get("/v1/models")
 async def get_models(
-    settings: Settings = Depends(get_settings),
+    llama: Llama = Depends(get_llama),
 ) -> ModelList:
-    assert llama is not None
     return {
         "object": "list",
         "data": [
             {
-                "id": settings.model_alias
-                if settings.model_alias is not None
-                else llama.model_path,
+                "id": model_alias,
                 "object": "model",
                 "owned_by": "me",
                 "permissions": [],
-            }
+            } for model_alias in llama
```

Comment:
This always returns the model alias, if no model_alias was configured, it defaults to the filename without extension

Reply:
I figured that was probably just my confiusion